### ISSUE TITLE
Workspace indicator (number) almost invisible small

### ISSUE THREAD

**Issue by l-t-k** [urob/komotray#3]
First of all, thanks for the great work! The application works like a charm.

The only thing is a real drawback for me at the moment is the size of the workspace indicator (the number in the taskbar icon). That is almost invisible small.

Is it possible to increase the size of it?

**Comment by urob** [comment]
Yes, the script shows whatever icons are stored inside the [`assets/icons`](https://github.com/urob/komotray/tree/main/assets/icons) directory. So you can just swap out the existing ones with whatever icons you like best. 

(Note that the size of the icon can't be changed as it is fixed by the taskbar. But you should be able to increase the numbers somewhat by dropping the surrounding frames.)
